client/state/selectors/is-site-checklist-complete.js

```diff
@@ -50,5 +52,14 @@ export default function isSiteChecklistComplete( state, siteId ) {
 		return false;
 	};
 
-	return siteChecklist.tasks.every( isTaskComplete );
+	const taskList = getTaskList( {
+		taskStatuses: siteChecklist.tasks,
+		phase2: siteChecklist.phase2,
+		siteVerticals: siteChecklist.verticals,
+		siteSegment: siteChecklist.siteSegment,
+	} ).getAll();
+	const taskUrls = getChecklistTaskUrls( state, siteId );
+	const hasNotUrl = task => ! ( task.id in taskUrls && ! taskUrls[ task.id ] );
```

Comment:
As we discussed earlier today, since this same logic is shared in https://github.com/Automattic/wp-calypso/blob/update/checklist-completion-check/client/my-sites/checklist/wpcom-checklist/component.jsx#L267, maybe there's a way to consolidate the logic in one place.